src/vivarium_gates_child_iv_iron/constants/models.py

```diff
@@ -10,29 +12,31 @@ def __new__(cls, value):
         return obj
 
 
+# noinspection PyPep8Naming
```

Comment:
This stuff should be added in a PR for the model implementation ticket, but it's fine for now. Just something to bear in mind for future model versions.